api/db-migrations/000009_ensembling_job.up.sql

```diff
@@ -22,6 +22,7 @@ CREATE TABLE IF NOT EXISTS ensembling_jobs
     retry_count      integer NOT NULL default 0,
     status           ensembling_job_status NOT NULL default 'pending',
     error            text,
+    dashboard_url    text,
```

Comment:
Do we need to persist this in the DB now? In a scenario where the dashboard URL is modified (although rare), we will have to update the data in the DB. For this reason, I think it is better to format the template string dynamically.

Reply:
I thought about it, it's quite troublesome because a few endpoints are affected:
1. create ensembling job
2. list ensembling job
3. get ensembling job

You might get a runtime performance hit (maybe negligible) when you call the list ensembling job method, since you have to construct the url for every job. I picked the persistence method because less code was required.

Reply:
But if you feel strongly about this we can always change it to a runtime thing. I think the data update might be relevant but ensembling jobs are a temporary thing, the dashboard wouldn't be very useful later on, i doubt it will be worth it to even update the values.

Reply:
actually after thinking about it, I'll change it to generate during runtime. so that its always independent of config.